client/src/components/subscription/subscriptionsList.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Subscription from './subscription';
+import UpdateSubscription from './updateSubscription';
+import CreateSubscription from './createSubscription';
+
+class SubscriptionsList extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      subscriptions: []
+    };
+
+    this.handleDelete = this.handleDelete.bind(this);
+  }
+
+  async componentDidMount() {
+    // When browser mounts subscriptionsList component, we make a call to get all their subscription
+    await fetch(`${window.location.pathname}/subscriptions`).then(data => data.json()).then(response => this.setState({ subscriptions: response }));
```

Comment:
i think we can break this up onto multiple lines for readability:

```suggestion
    await fetch(`${window.location.pathname}/subscriptions`)
      .then(data => data.json())
      .then(response => this.setState({ subscriptions: response }));
```